server/server.go

```diff
@@ -1465,7 +1494,7 @@ func (s *Server) campaignLeader() {
 	})
 
 	CheckPDVersion(s.persistOptions)
-	log.Info("PD cluster leader is ready to serve", zap.String("pd-leader-name", s.Name()))
+	log.Info("leader is ready to serve", zap.String("leader-name", s.Name()))
```

Comment:
If we know the mode, we can log it by the log. sometimes, etcd also log some log similar `leader is`, it's may confuse.